packages/app-accounts/src/modals/Import.tsx

```diff
@@ -9,10 +9,10 @@ import { ModalProps } from '../types';
 
 import React from 'react';
 import { AddressRow, Button, InputAddress, InputFile, Modal, Password, TxComponent } from '@polkadot/react-components';
-import { isHex, isObject, u8aToString } from '@polkadot/util';
+import { isObject, u8aToString } from '@polkadot/util';
 import keyring from '@polkadot/ui-keyring';
-
 import translate from '../translate';
+import {updateToV3KeyringFormat} from "@polkadot/react-api/util/keyringFormat";
```

Comment:
How can this newly added function be part of polkadot?

Reply:
Since this is a monorepo.. I added the new function in @polkadot/react-api repo's util.. so that it can be re-used